packages/vite/src/node/server/__tests__/pluginContainer.spec.ts

```diff
@@ -10,6 +10,38 @@ let resolveId: (id: string) => any
 let moduleGraph: ModuleGraph
 
 describe('plugin container', () => {
+  it('has bound methods', async () => {
+    expect.assertions(14)
+    const entryUrl = '/x.js'
+
+    const plugin: Plugin = {
+      name: 'p1',
+      load(id) {
+        // bound functions and bound arrow functions do not have prototype
+        expect(this.load.prototype).equals(undefined)
+        expect(this.parse.prototype).equals(undefined)
+        expect(this.resolve.prototype).equals(undefined)
```

Comment:
```suggestion
        expect(this.parse.prototype).equals(undefined)
        expect(this.resolve.prototype).equals(undefined)
        expect(this.load.prototype).equals(undefined)
```
Let's reorder this as well.